### PR TITLE
Add dashboard in iframe

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/app/entities/flair-bi/flair-bi-content-header.controller.js [hunk 1]
```diff
@@ -125,6 +127,7 @@
         vm.changeViewAndUpdateDashboard = changeViewAndUpdateDashboard;
         vm.dashboardId = $stateParams.dashboardId;
         vm.viewId = $stateParams.id;
+        vm.isIframe = $stateParams.isIframe==="true"?true:false;
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/flair-bi-content-header.controller.js:130]: create a function for this

File: src/main/webapp/app/entities/flair-bi/flair-bi-content-header.controller.js [hunk 2]
```diff
@@ -333,7 +337,7 @@
             var temporalDataTypes = filterParametersService.getComparableDataTypes();
             selectedBookmark.featureCriteria.forEach(function (criteria) {
                 var isTemporal = criteria.dateRange && temporalDataTypes.indexOf(criteria.feature.type.toLowerCase()) > -1;
-                filter[criteria.feature.name] = applyFeatureCriteria(isTemporal, criteria.metaData, criteria.feature, criteria.value, criteria.feature.name);  
+                filter[criteria.feature.name] = applyFeatureCriteria(isTemporal, criteria.metaData, criteria.feature, criteria.value, criteria.feature.name);
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/flair-bi-content-header.controller.js:340]: what is the change in this line?

File: src/main/webapp/app/entities/flair-bi/flair-bi-content-header.controller.js [hunk 3]
```diff
@@ -883,5 +887,28 @@
             if (containerColor)
                 $('.page-wrapper-full-screen').css('background-color', containerColor)
         }
+
+        function share() {
+            $uibModal
+                .open({
+                    animation: true,
+                    templateUrl:
+                        "app/entities/flair-bi/share-dialog/share-dialog.html",
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/flair-bi-content-header.controller.js:896]: rename html and controller to ShareDialogController to SharedLinkDialogController and share-dialog.html to shared-link-dialog.html

**Pull request merged**